src/io/flutter/view/InspectorPerfTab.java

```diff
@@ -42,10 +55,34 @@
 
     add(Box.createVerticalStrut(6));
 
+      // TODO(terry): Add Basic line chart Memory View
+    //JPanel memChart = FlutterLineChart.createJPanelView(parentDisposable, app);
+    //add(memChart, BorderLayout.NORTH);
+
+// TODO(terry): How to get an IdeServices???
+// TODO(terry): Renable when using below
+    FlutterStudioProfilers fsp = new FlutterStudioProfilers(parentDisposable, app);
+    FlutterStudioProfilersView view = new FlutterStudioProfilersView(fsp);
+    add(view.getComponent(), BorderLayout.CENTER);
+
+    // TODO(terry): Basic linechart test.
+//    JPanel basicChart = FlutterLineChart.createJPanelView(parentDisposable, app);
+//    add(basicChart, BorderLayout.NORTH);
+
+    // TODO(terry): Flutter Memory View.
+//    JPanel memoryView = FlutterMemoryView.createJPanelView(parentDisposable, app);
+//    add(memoryView, BorderLayout.NORTH);
+
+/*
```

Comment:
don't comment this code out. it should still apply for now when 
if (!FlutterSettings.getInstance().isMemoryProfiler())